indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -436,6 +437,28 @@ def can_review(self, user):
         else:
             return False
 
+    def is_convener(self, user):
+        if not user:
+            return False
+        elif not self.event_new.can_manage(user, role='track_convener'):
+            return False
+        elif self.event_new in user.global_convener_for_events:
+            return True
+        elif user.convener_for_tracks & self.reviewed_for_tracks:
+            return True
+        else:
+            return False
+
+    def can_judge(self, user):
+        if not user:
+            return False
+        elif self.event_new.can_manage(user):
+            return True
+        elif self.is_convener(user):
```

Comment:
`is_convener` is a more expensive check than `allow_convener_judgment`, so I would flip the order.